app/src/main/java/com/rob729/quiethours/util/StoreSession.kt

```diff
@@ -30,6 +32,16 @@ object StoreSession {
         return sharedPreferences.getInt(key, 0)
     }
 
+    fun setNightMode(key: String, value: Boolean) {
```

Comment:
when you already know that this function will be used to set night mode then take the key as a function parameter?